org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/comparison/StructureDefinitionComparer.java

```diff
@@ -390,12 +392,11 @@ private boolean compareChildren(ProfileComparison comp, StructuralMatch<ElementD
   }
 
 
-  private boolean compareDiff(String path, String sliceName, DefinitionNavigator left, DefinitionNavigator right, ProfileComparison res, Base parent) throws DefinitionException, FHIRFormatError, IOException {
+  private boolean compareDiff(String path, String sliceName, DefinitionNavigator left, DefinitionNavigator right, ProfileComparison res, List<Base> parents) throws DefinitionException, FHIRFormatError, IOException {
     assert(path != null);  
     assert(left != null);
     assert(right != null);
     assert(left.path().equals(right.path()));

```

Comment:
Based on other methods returning `parents.get(0)`, should we also have an assert of `parents.size() > 0`?

Reply:
well ok, but it could never be empty